manifests/t/TortoiseSVN/TortoiseSVN/1.14.29465/TortoiseSVN.TortoiseSVN.installer.yaml

```diff
@@ -0,0 +1,38 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=QUSU.CRLF.5-1-25206-1000.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: TortoiseSVN.TortoiseSVN
+PackageVersion: 1.14.29465
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://mirrors.gigenet.com/OSDN//storage/g/t/to/tortoisesvn/1.14.5/Application/TortoiseSVN-1.14.5.29465-x64-svn-1.14.2.msi
```

Comment:
I think this one is failing the domain validation. This should probably be the osdn.net urls like others.